SimMuon/GEMDigitizer/src/ME0PreRecoGaussianModel.cc

```diff
@@ -37,14 +37,20 @@ ME0PreRecoGaussianModel::ME0PreRecoGaussianModel(const edm::ParameterSet& config
   minBunch_(config.getParameter<int>("minBunch")), 
   maxBunch_(config.getParameter<int>("maxBunch")),
   instLumi_(config.getParameter<double>("instLumi")),
-  rateFact_(config.getParameter<double>("rateFact"))
+  rateFact_(config.getParameter<double>("rateFact")),
+  referenceInstLumi_(config.getParameter<double>("referenceInstLumi"))
 {
   // polynomial parametrisation of neutral (n+g) and electron background
   // This is the background for an Instantaneous Luminosity of L = 5E34 cm^-2 s^-1
-  neuBkg.push_back(899644.0);     neuBkg.push_back(-30841.0);     neuBkg.push_back(441.28);
-  neuBkg.push_back(-3.3405);      neuBkg.push_back(0.0140588);    neuBkg.push_back(-3.11473e-05); neuBkg.push_back(2.83736e-08);
-  eleBkg.push_back(4.68590e+05);  eleBkg.push_back(-1.63834e+04); eleBkg.push_back(2.35700e+02);
-  eleBkg.push_back(-1.77706e+00); eleBkg.push_back(7.39960e-03);  eleBkg.push_back(-1.61448e-05); eleBkg.push_back(1.44368e-08);
+  neuBkg.push_back(0.00386257);
```

Comment:
however, this approach does not lend itself to any sort of future proofing (eg, when your asked to evaluate a range of lumis..)